src/routes/LogInForm.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import users from "../localDB/users.json";
+import { useNavigate } from "react-router-dom";
+import "./login.css";
+
+export default function LogInForm() {
+  const [usersData, setUsersData] = React.useState(users);
+  const navigate = useNavigate();
+
+  const [inputPassword, setInputPassword] = React.useState("");
+  const [inputEmail, setInputEmail] = React.useState("");
```

Comment:
In order to not use React.useState you can just import the useState from react library.